### PR TITLE
feat: added deletion confirmation for connections

### LINKED ISSUE #18: connection deletion confirmation
we need to add confirmation of connection deletion

### PR DESCRIPTION
**Pull request by AlexEbenrode**
- Added deletion confirmation for connections
- Fix listing bucket for connections without specified bucket

### CODE REVIEW

File: lib/pages/settings_page/settings_page.dart [hunk 1]
```diff
@@ -22,7 +22,29 @@ class SettingsPage extends StatelessWidget {
...
+                      Button(
+                        onPressed: () {
+                          storage.deleteConnection(connection);
+                          Navigator.pop(context, 'User deleted connection');
+                        },
+                        child: const Text('Delete'),
+                      ),
+                      Button(
+                        onPressed: () {
+                          Navigator.pop(context, 'User canceled dialog');
```
Comment (goshander) [inline lib/pages/settings_page/settings_page.dart]: Second parameter is cancel dialog meta, maybe it was short word or several words with underscore as a key in separate constants file

**Review by goshander — CHANGES_REQUESTED** [review]

File: lib/pages/settings_page/settings_page.dart (hunk 1 above)
Comment (AlexEbenrode) [reply lib/pages/settings_page/settings_page.dart]: i dont understand what you want change...

Comment (AlexEbenrode) [reply lib/pages/settings_page/settings_page.dart]: i just copy example from here https://bdlukaa.github.io/fluent_ui/#/popups/content_dialog

Comment (AlexEbenrode) [reply lib/pages/settings_page/settings_page.dart]: well maybe do it with general localizaton?

**Review by goshander — APPROVED** [review]

**Comment by goshander** [comment]
:tada: this issue has been resolved in version 1.0.0 :tada:

release is available on [GitHub](<github_release_url>)

**Pull request merged**